### ISSUE TITLE
Random drop some nodes when training

### ISSUE THREAD

**Issue by logsea** [rusty1s/pytorch_geometric#1201]
## ❓ Questions & Help

Hi, I want to drop some nodes randomly at the begining of the model, and I find `dropout_adj` can do this on edges. So if there have any good way to do this on node?

**Comment by rusty1s** [comment]
Two options to achieve that:
1. Sample nodes and use `torch_geometric.utils.subgraph` at the beginning of the model.
2. Use the normal PyTroch `dropout` function, e.g.,
```python
p = F.dropout(x.new_ones((x.size(0), 1), p=0.5, training=self.training)
x = p * x
```

**Comment by logsea** [comment]
I use the first option and it can solve my problem! Thank you for the solution.
